backslash/contrib/slash_plugin.py

```diff
@@ -76,6 +77,7 @@ def __init__(self, url=None, keepalive_interval=None, runtoken=None,
         self._error_containers = {}
         self._runtoken = runtoken
         self._propagate_exceptions = propagate_exceptions
+        self._ended = False
```

Comment:
This might cause an attempt at cleanup even before session is assigned. It should be set to `True` only after the session is reported as started